app/_assets/javascripts/app.js

```diff
@@ -336,16 +336,84 @@ jQuery(function () {
     ); // Adjust scroll speed
   }
 
+
+// Plugin filter all / only available
+var pluginsAllOrOnly = "all";
+function showPluginBanner(target){
+  var hubCards = $(".hub-cards");
+  if (!hubCards.length){
+    return;
+  }
+
+  var allOnlyToggle = $('#plugin-banner');
+  if (target == "all" || target == "open-source"){
+    allOnlyToggle.hide();
+    return;
+  }
+
+  allOnlyToggle.show();
+
+  if (!allOnlyToggle.length){
+    allOnlyToggle = $('<div id="plugin-banner" class="content" style="padding:0"><blockquote class="note"></blockquote></div>');
+    hubCards.prepend(allOnlyToggle)
+  }
+
+  var tierMap = {
+    "plus": "Plus",
+    "ee-compat": "Enterprise"
+  }
+
+  var content = allOnlyToggle.find("blockquote");
+
+  var other = pluginsAllOrOnly == "all" ? "only" : "all";
+  content.html("Showing <strong>"+pluginsAllOrOnly+"</strong> plugins available on the " + tierMap[target] + " tier. <u>Switch to show <strong>"+other+"</strong> " + tierMap[target] + " plugins?</u>")
```

Comment:
For Enterprise plugins: we're kind of stuck in awkward place, where the on-prem Enterprise tier includes all Plus and Enterprise plugins. Since there's no separate Plus tier for Kong Gateway alone, the filter that selects Enterprise _only_ isn't accurate for that group. A couple of approaches we could take:

* Have two Enterprise filters: Konnect Enterprise and Gateway Enterprise
* Label the "Subscription tiers" section as "Konnect subscription tiers" only
* Have two links in the Enterprise banner, something like: 

  >  Showing plugins only available on the Konnect Enterprise tier. Switch to show Kong Gateway Enterprise plugins, or all Enterprise plugins?

Now that I look again, I'm not really a fan of options 2 and 3. So maybe we need another filter.

Reply:
I've added a second filter when you open up `Enterprise` to enable showing plus too. It's not ideal, but I prefer it there to splitting on-prem and cloud enterprise in to separate filters at this point. I'd like to keep the changes to a minimum